backend/controllers/books.js

```diff
@@ -0,0 +1,19 @@
+const Book = require('../models/Books');
+
+module.exports.getBooks = async function (req, res) {
+
+    try {
+        const books = await Book.find();
+        res.status(200).json(books);
+    } catch (e) {
+
```

Comment:
Add some responce on error. Set some 'internal error' status code and send some message to user, like: 'Can't find books. Something went wrong'